app/assets/javascripts/table_floating_header.js

```diff
@@ -1,147 +0,0 @@
-var floating_header = function() {
```

Comment:
Are we certain this function for tables isn't used anywhere else in the codebase?

Reply:
ill give this a 2nd pass i couldnt locate any on the first trip